### ISSUE TITLE
two bugs and a feature request

### ISSUE THREAD

**Issue by jgrjgr** [woodsshin/UPNPSample#1]
Hey ,  

re the feature request and a two bugs

the two bugs: 

- upnp description not being set on the routers upnp table which means i can't put the game name in there so if anyone looks at the table there is no application identity for it
- lease time gets set to zero which is the more concerning one as it leave the port open forever, the other problem is that some** routers will either set a default value which is likely to timeout in a short period of time or simple reject or ignore the request all together

the feature request :  can we have a string output for the "control url" so we can construct out own upnp requests against the router as some of them have storage attached as well

**Comment by woodsshin** [comment]
Hey thanks for your feedback. 
The reason why control url wasn't exposed to blueprint is we don't know which url is right one on your router. Of course, you know your own router. But when you release your game, most of people don't know well about it. Let me think about it.

**Comment by jgrjgr** [comment]
fair enough, it's only useful to a small number of people,  the main problem is that description and lease time more so the lease time as it's not always the case that the engine gets shutdown formally

**Comment by jgrjgr** [comment]
![image](https://github.com/woodsshin/UPNPSample/assets/7400269/bc31691b-f032-4803-8585-3e095d6b9273)

![image](https://github.com/woodsshin/UPNPSample/assets/7400269/dffa072a-3da8-4445-a7a1-38beb1322cf9)

**Comment by jgrjgr** [comment]
everything else seems to work, except the "PNPinfo Desc" , "PNPinfo Duration" don't appear to work as intended
